public/components/common/SignInControl.tsx

```diff
@@ -77,6 +78,23 @@ export const SignInControl: React.FunctionComponent<SignInControlProps> = (props
               }
             />
           </Form>
+          {!props.useEmail && (
+            <p className="text-red-700 mt-1">
+              <Trans id="signin.message.onlyadmins">Currently only allowed to sign in to an administrator account</Trans>
+            </p>
+          )}
+        </div>
+      ) : (
+        <div>
+          <p className="text-muted">
+            <Trans id="signin.message.emaildisabled">
+              Email authentication has been disabled by an administrator. If you have an administrator account and need to bypass this restriction, please{" "}
+              <span className="text-bold clickable" onClick={() => setUseEmail(true)}>
```

Comment:
I think we could turn this into a `<a href="#" className="text-link"` ... instead of a clickable span. I'm slowing removing all occurrences of that because of accessibility issues.

Reply:
No problem, I see, I changed it!